src/entity/posts/utils.ts

```diff
@@ -386,30 +386,28 @@ export const createSharePost = async ({
       },
     } as DeepPartial<SharePost>);
 
-    if (ctx) {
-      const vordrStatus = await checkWithVordr(
-        {
-          id: createdPost.id,
-          content: createdPost.title || undefined,
-          type: VordrFilterType.Post,
-        },
-        { con, userId, req: ctx.req },
-      );
-
-      if (vordrStatus) {
-        createdPost.banned = true;
-        createdPost.showOnFeed = false;
+    const vordrStatus = await checkWithVordr(
+      {
+        id: createdPost.id,
+        content: createdPost.title || undefined,
+        type: VordrFilterType.Post,
+      },
+      { con, userId, req: ctx?.req },
+    );
```

Comment:
What is the goal here, because this is only for shared posts right? 

Would it make sense to move it to approval worker `api.source-post-moderation-approved-notification` so it catches all posts approved after moderation?

Reply:
I think Ido also wants it to happen pre-submission: https://dailydotdev.slack.com/archives/C013RGVGPN1/p1732728195138469?thread_ts=1732726519.529059&cid=C013RGVGPN1

Reply:
Exactly. we should silently ignore it

Reply:
Yeah, it could've been moved to that worker. But it should also happen before it reaches moderation too

Reply:
Still, I think we can push this PR. If in case the user was vordred after the submission which won't have the flag yet. Edge casey but at least a second level filter.

Reply:
@sshanzel's PR #2497 handles it so that it blocks it from reaching moderation